docs/installation.rst

```diff
@@ -11,7 +11,7 @@ Prerequisites
 
 Before you are ready to run Saleor you will need certain software installed on your computer.
 
-#. `Python <https://www.python.org/>`_ version 3.5.x or 2.7.x
+#. `Python <https://www.python.org/>`_ version 3.5.x
```

Comment:
Currently, we support Python 3.4, 3.5 and 3.6 (CI is configured to test the codebase against these), so instead 3.5.x we could write "Python version 3.4 or higher".

Reply:
Also, the second item below about `pip` could be dropped at all, I think. According to docs `pip` comes with Python by default since version 3.4.